flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -108,10 +109,16 @@ protected Cluster buildCluster(Cluster.Builder builder) {
 	private static final String SELECT_DATA_QUERY = "SELECT * FROM flink.test;";
 
 	private static final ArrayList<Tuple3<String, Integer, Integer>> collection = new ArrayList<>(20);
+	private static final ArrayList<org.apache.flink.types.Row> rowCollection = new ArrayList<>(20);
```

Comment:
I see, In that case i would suggest to import the Flink for type and use the qualified name for the Datastax row.